src/App/Header/NavDropdown.tsx

```diff
@@ -45,7 +51,7 @@ export default function NavDropdown({ links }: Props) {
             leaveFrom="opacity-100 translate-y-0"
             leaveTo="opacity-0 translate-y-1"
           >
-            <Popover.Panel className="absolute right-0 z-10 mt-3 w-screen max-w-56 shadow-[0px_8px_20px] shadow-gray-d1 rounded-lg bg-gray-l6 dark:bg-blue-d6 transform">
+            <Popover.Panel className="absolute right-0 z-10 mt-3 sm:w-screen max-w-[300px] shadow-[0px_8px_20px] shadow-gray-d1 rounded-lg bg-gray-l6 dark:bg-blue-d6 transform">
```

Comment:
```suggestion
            <Popover.Panel className="absolute right-0 z-10 mt-3 sm:w-screen max-w-xs shadow-[0px_8px_20px] shadow-gray-d1 rounded-lg bg-gray-l6 dark:bg-blue-d6 transform">
```

or 

```suggestion
            <Popover.Panel className="absolute right-0 z-10 mt-3 sm:w-screen max-w-80 shadow-[0px_8px_20px] shadow-gray-d1 rounded-lg bg-gray-l6 dark:bg-blue-d6 transform">
```

if `300px` isn't strict, we could opt to these util classes (`320px`)

Reply:
It's not strict. Since there's no real designs at play here, there's wiggle room in the final implementation, as long as it captures the requirements to have everything in a single dropdown menu. :+1:

Reply:
Fixed in [1bdd899](https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2815/commits/1bdd899a3cda0ae9e5c11e584c17c60534276185)